coord/src/module.c

```diff
@@ -1625,7 +1618,7 @@ static int searchResultReducer(struct MRCtx *mc, int count, MRReply **replies) {
   }
 
   searchRequestCtx_Free(req);
-  RS_CHECK_FUNC(RedisModule_BlockedClientMeasureTimeEnd, bc);
```

Comment:
I guess this change is because we are not supporting older Redis versions anymore?

Reply:
Right. This was added in 6.0.9, and on master, we support only >6 redis